ZelBack/src/services/explorerService.js

```diff
@@ -644,7 +646,15 @@ async function processBlock(blockHeight, isInsightExplorer) {
     if (!syncStatus.data.synced) {
       setTimeout(() => {
         processBlock(blockHeight, isInsightExplorer);
-      }, 2 * 60 * 1000);
+      }, 2 * MINUTE_IN_MS);
+      return;
+    }
+    const benchmarkBenchRes = await benchmarkService.getStatus();
```

Comment:
this call does not have any caching. My worry is that it will be executed very oftenly and can be heavy for the benchmark app side. Slowing everything down with high usage. Can we recheck this worry and test the usage or/and adding some 30 sec cache for the call?

Reply:
I add a Cache for 1 minute.